src/features/broker-protection/actions/extract.js

```diff
@@ -250,21 +233,34 @@ function scrapedDataMatchesUserData (userData, scrapedData) {
  * @param {Record<string, any>} profile
  */
 export function aggregateFields (profile) {
-    const addressCityStateArray = profile.addressCityState || []
-    const addressCityStateListArray = profile.addressCityStateList || []
-    const addresses = [...new Set([...addressCityStateArray, ...addressCityStateListArray])]
-
+    // addresses
+    const combinedAddresses = [
+        ...profile.addressCityState || [],
+        ...profile.addressCityStateList || [],
+        ...profile.addressFullList || [],
+        ...profile.addressFull || []
+    ]
+    const addressMap = new Map(combinedAddresses.map(addr => [`${addr.city},${addr.state}`, addr]))
+    const addresses = [...addressMap.values()]
```

Comment:
we can now report all found addresses, since we know they'll be valid if we got this far. The Map helps to de-dupe. I avoided JSON.stringify since the ordering of the keys could affect it.